### PR TITLE
Document how to restart failed CI jobs.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation

Give maintainers hints on how to deal with failed jobs that look spurious.

### CODE REVIEW

File: dev/doc/MERGING.md [hunk 1]
```diff
@@ -76,6 +76,16 @@ documentation and test cases. If not, they should leave a comment on the PR and
 put the approriate label. Otherwise, they are expected to merge the PR using the
 [merge script](/dev/tools/merge-pr.sh).
 
+When CI has a few failures which look spurious, restarting the corresponding
+jobs is a good way of ensuring this was indeed the case.
+To restart a job on Travis, you should connect using your GitHub account;
+being part of the Coq organization on GitHub should give you the permission
+to restart a job on Travis.
```
Comment (SkySkimmer) [inline dev/doc/MERGING.md]: >to restart a job on Travis

This phrase is repeated at the beginning and end of the sentence, it sounds like an echo.

Comment (Zimmi48) [reply dev/doc/MERGING.md]: Indeed, thanks.

**Comment by Zimmi48** [comment]
Now based on #7800 to avoid conflicts.

**Review by gares — APPROVED** [review]
OK wrt test-suite and merging

**Review by ejgallego — APPROVED** [review]

**Pull request merged**